addons/product/models/product_pricelist.py

```diff
@@ -332,3 +337,21 @@ def _unlink_except_used_as_rule_base(self):
                 '\n'.join(linked_items.base_pricelist_id.mapped('display_name')),
                 '\n'.join(linked_items.pricelist_id.mapped('display_name'))
             ))
+
+    # Tracking methods
+    def _mail_track(self, tracked_fields, initial_values):
+        changes, tracking_value_ids = super()._mail_track(tracked_fields, initial_values)
+        if len(changes) > len(tracking_value_ids):
+            for changed_field in changes:
+                if tracked_fields[changed_field]['type'] in ['one2many', 'many2many']:
+                    field = self.env['ir.model.fields']._get(self._name, changed_field)
+                    vals = {
+                        'field': field.id,
+                        'field_desc': field.field_description,
+                        'field_type': field.ttype,
+                        'tracking_sequence': field.tracking,
+                        'old_value_char': ', '.join(initial_values[changed_field].mapped('name')),
+                        'new_value_char': ', '.join(self[changed_field].mapped('name')),
+                    }
+                    tracking_value_ids.append(Command.create(vals))
+        return changes, tracking_value_ids
```

Comment:
Conclusion from PM:
move this to mail instead (probably in `create_tracking_values`), so that mail will now also support `one2many` and `many2many` and get rid of these lines in both `accounting` and `project` too. =]